projects/storefrontstyles/scss/components/cart/_applied-coupons.scss

```diff
@@ -29,15 +29,32 @@
     border-radius: $border-radius;
   }
 
+  .cx-coupon-card-grid {
+    @include media-breakpoint-down(md) {
+      flex: 0 0 100%;
+      max-width: 100%;
+    }
+
+    @include media-breakpoint-up(md) {
+      padding-right: 0;
+    }
+  }
+
   .textonly {
     margin: 0 10px 10px 0;
     font-size: 14px;
   }
 
   .cx-coupon-apply {
+    display: inline-flex;
+    align-items: center;
     @include type('6');
     padding: 1rem 1rem;
     align-items: flex-start;

```

Comment:
align-items center being overriden by this, and I think we need center, so remove this

Reply:
Can't find the .close class, but it should have a padding because its near the text